src/main/java/com/google/sps/servlets/JoinRoomServlet.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that takes in the user's inputted Room ID and directs them
+    to the associated private room once ensurred that it's an actual room*/
+@WebServlet("/join-room")
+public final class JoinRoomServlet extends HttpServlet {
+  private String inputtedUserTag = "user-party-link";
+  private String currentRoomId = "";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // TODO: Create hard-coded hashmap of {Room ID : URL} Hashmap
+    // TODO: Print json-ified string to /join-room page for private room to fetch
+  }
+
+	@Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+		// Take in user's private room and store it to global variable
+		currentRoomId = request.getParameter(inputtedUserTag);
+		response.getWriter().println(currentRoomId);
+		// TODO: Redirect to private room Page
+		response.sendRedirect("/views/private-room.html");
```

Comment:
Create a global variable in a util file that contains this html path. Then use it here.

Useful if you ever change the path to this html file in the future.